x-pack/plugins/cases/public/components/case_view/case_view_metrics.tsx

```diff
@@ -6,35 +6,280 @@
+  const creationDate = getMaybeDate(date);
+  if (!creationDate.isValid()) {
+    return getEmptyTagValue();
+  }
+
+  return (
+    <FormattedRelativePreferenceDate
+      data-test-subj={'case-metrics-lifespan-creation-date'}
+      value={date}
+    />
+  );
+};
+
+const getInProgressDuration = (duration: number) => {
+  if (duration <= 0) {
+    return getEmptyTagValue();
+  }
+
+  return formatDuration(duration);
+};
+
+const formatDuration = (milliseconds: number) => {
+  return prettyMilliseconds(milliseconds, { compact: true, verbose: true });
+};
+
+const getOpenCloseDuration = (openDate: string, closeDate: string | null): string | undefined => {
+  if (closeDate == null) {
+    return;
+  }
+
+  const openDateObject = getMaybeDate(openDate);
+  const closeDateObject = getMaybeDate(closeDate);
+
+  if (!openDateObject.isValid() || !closeDateObject.isValid()) {
+    return;
+  }
+
+  return formatDuration(closeDateObject.diff(openDateObject));
+};
+
+const Title = euiStyled(EuiFlexItem)`
+  font-size: ${({ theme }) => theme.eui.euiSizeM};
+  font-weight: bold;
+`;
+
+const CaseStatusMetricsItem: React.FC<{
+  title: string;
+  value: JSX.Element | string;
+}> = React.memo(({ title, value }) => (
+  <EuiFlexGroup direction="column" gutterSize="s" responsive={false}>
+    <Title>{title}</Title>
+    <EuiFlexItem>{value}</EuiFlexItem>
+  </EuiFlexGroup>
+));
+CaseStatusMetricsItem.displayName = 'CaseStatusMetricsItem';
+
+const CaseStatusMetricsOpenCloseDuration: React.FC<{
+  title: string;
+  value?: string;
+  reopens: string[];
+}> = React.memo(({ title, value, reopens }) => {
+  const valueText = getOpenCloseDurationText(value, reopens);
+
+  return (
+    <EuiFlexGroup direction="column" gutterSize="s" responsive={false}>
+      <Title>{title}</Title>
+      {value != null && caseWasReopened(reopens) ? (
+        <ValueWithExplanationIcon value={valueText} explanationValues={reopens} />
+      ) : (
+        <EuiFlexItem>{valueText}</EuiFlexItem>
+      )}
+    </EuiFlexGroup>
+  );
+});
+CaseStatusMetricsOpenCloseDuration.displayName = 'OpenCloseDuration';
+
+const getOpenCloseDurationText = (value: string | undefined, reopens: string[]) => {
+  if (value == null) {
+    return getEmptyTagValue();
+  } else if (reopens.length > 0) {
+    return `${value} ${CASE_REOPENED}`;
+  }
 
-const MetricValue = styled(EuiFlexItem)`
+  return value;
+};
+
+const caseWasReopened = (reopens: string[]) => {
+  return reopens.length > 0;
+};
+
+const ValueWithExplanationIcon: React.FC<{
+  value: string | JSX.Element;
+  explanationValues: string[];
+}> = React.memo(({ value, explanationValues }) => {
+  const content = (
+    <>
+      {CASE_REOPENED_ON}
+      {explanationValues.map((explanationValue, index) => (
+        <React.Fragment key={`explanation-value-${index}`}>
+          <FormattedDate
+            data-test-subj={`case-metrics-lifespan-reopen-${index}`}
+            value={explanationValue}
+          />{' '}
```

Comment:
Do I need to translate a space? Also open to better ideas for separating the dates.

Reply:
I am not sure this space will render correctly in all cases. It looks like you need to write some css to separate things correctly.